### ISSUE TITLE
simple escaper  performance

### ISSUE THREAD

**Issue by aravorn6** [tipsy/j2html#45]
I got a worse performance with v0.88 

```
                long startTrainTime = System.currentTimeMillis();

		EmptyTag c =	img().withClass("image").withSrc(sb.toString());
	
		long endTrainTime = System.currentTimeMillis();
		DecimalFormat df = new DecimalFormat("#.#####");
		System.out.println(df.format((endTrainTime - startTrainTime) / 1000. / 60.));
```
v.0.7: 0,00007
v.0.88: 0,06715

looks like the last simple escaper change

**Comment by tipsy** [comment]
Wow, that's horrible. I went for a pretty slow implementation because of readability, but I guess I have to rewrite it.

**Comment by tipsy** [comment]
@aravorn6 Fixed in 29e16df0576f33c86d529643dc1cc965e90fc89c

```
SimpleEscaper#short: 3ms
SimpleEscaper#long:  45ms
ApacheEscaper#short: 35ms
ApacheEscaper#long:  202ms
```

**Comment by tipsy** [comment]
@aravorn6 0.99 is now out.
